hazelcast/src/test/java/com/hazelcast/internal/serialization/impl/SerializerHookLoaderTest.java

```diff
@@ -0,0 +1,70 @@
+package com.hazelcast.internal.serialization.impl;
+
+import com.hazelcast.config.SerializationConfig;
+import com.hazelcast.config.SerializerConfig;
+import com.hazelcast.nio.serialization.HazelcastSerializationException;
+import com.hazelcast.test.HazelcastParallelClassRunner;
+import com.hazelcast.test.HazelcastTestSupport;
+import com.hazelcast.test.annotation.ParallelTest;
+import com.hazelcast.test.annotation.QuickTest;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.junit.runner.RunWith;
+
+import java.util.Map;
+
+import static org.junit.Assert.assertNotNull;
+
+/**
+ * SerializerHookLoader Tester.
+ *
+ */
+@RunWith(HazelcastParallelClassRunner.class)
+@Category({QuickTest.class, ParallelTest.class})
+public class SerializerHookLoaderTest extends HazelcastTestSupport {
+
+    @Test
+    public void testLoad() throws Exception {
+        ClassLoader classLoader = getClass().getClassLoader();
```

Comment:
I would put the classloader in a private field and. Now every tests gets longer because of this, even though they all share the same logic.